wiki/Guides/BSS_issues/de.md

```diff
@@ -4,39 +4,58 @@ tags:
   - beatmap submission system issues
   - Ich kann meine eigenen Beatmaps nicht einreichen!
   - Probleme mit dem Beatmap Submission System
-outdated_translation: true
-outdated_since: 325c441d7a67e45daa10ec8d91295e17c1f54999
 ---
 
-# Wie man Probleme mit dem BSS löst
+# Wie man häufige Probleme mit dem BSS löst
 
-Wenn das **[Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)** (***BSS***) dich nicht deine eigenen Beatmaps einreichen lässt, weil es der Meinung ist, dass die Beatmap nicht dir gehört, dann mache folgendes:
+Beim Einreichen einer Beatmap zeigt das **[Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)** (***BSS***) möglicherweise bestimmte Warnungen an und hindert dich daran, deine Beatmap hochzuladen. Dieser Artikel hilft dir dabei, diese Warnungen zu verstehen und die Probleme zu beheben.
+
+## Du bist nicht der Besitzer der Beatmap, die du gerade hochladen möchtest!
+
+::: Infobox
+![Screenshot der Warnung](img/errormessage-DE.png "Die Warnung im Editor")
+:::
+
+Diese Warnung tritt beim Mapping vermutlich am meisten auf und wird oft durch einen falsch gesetzten Parameter in der [.osu-Datei](/wiki/Client/File_formats/osu_(file_format)), einen fehlerhaften Submission-Cache oder eine einfache Namensänderung verursacht. Mache Folgendes, um das Problem zu lösen:
 
 1. Gehe zum Ordner der Beatmap.
-   - Du kannst ihn leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
-2. Beende osu! (Dadurch wird sichergestellt, dass alle Änderungen korrekt übernommen werden.)
+   - Du kannst ihn leicht erreichen, indem du im [Beatmap-Editor](/wiki/Client/Beatmap_editor) `Datei` und dann `Songs-Ordner öffnen` anklickst.
+2. Beende osu!. Dadurch wird sichergestellt, dass alle Änderungen korrekt übernommen werden.
 3. Öffne die `.osu`-Dateien für die Schwierigkeitsstufen mit irgendeinem Texteditor (z. B. Notepad).
-4. Gib deinen Benutzernamen bei `Creator` an. (Wenn dein Name Leerzeichen enthält, dann nutze Leerzeichen.)
+4. Stelle sicher, dass dein Benutzername bei `Creator` bereits richtig geschrieben wird. Ist dies nicht der Fall, schreibe deinen aktuellen Benutzernamen hin.
 5. Setze `BeatmapID` auf `0`.
 6. Setze `BeatmapSetID` auf `-1`.
-7. Stelle sicher, dass der Name des Beatmap-Ordners nicht mit einer Zahlenfolge beginnt.
-   - Z. B. `1000 - Songname` sollte `Songname` sein.
-8. Lösche die Datei(en) im Ordner `SubmissionCache`. Dieser befindet sich im osu!-Installationsordner.
+7. Stelle sicher, dass der Name des Beatmap-Ordners nicht mit einer Zahlenfolge beginnt. Ansonsten ändere den Namen entsprechend ab.
+   - Z. B. `1000 - Songname` sollte `ABCDE - Songname` sein.
+8. Lösche alle Dateien im Ordner `SubmissionCache`. Standardmäßig befindet sich dieser im Verzeichnis `Data` des osu!-Installationsordners.
```

Comment:
```suggestion
8. Lösche alle Dateien im Ordner `SubmissionCache`. Standardmäßig befindet sich dieser im versteckten Verzeichnis `Data` des osu!-Installationsordners.
```
The "hidden" in the original article ("this folder is **hidden** inside the `Data` directory") refers to the fact that `Data` is a hidden directory, which is quite an important detail.